### ISSUE TITLE
 Issues Importing Custom Robot Model from SolidWorks into Genesis

### ISSUE THREAD

**Issue by juanjuanjuanbing** [Genesis-Embodied-AI/Genesis#164]
Thank you for your exciting work! I have encountered some issues while trying to import a custom robot model into Genesis. Here is a detailed description of the problem:

The robot was modeled in SolidWorks and exported in URDF format using a SolidWorks plugin. This URDF file was then converted to an MJCF file using the Mujoco engine. However, the resulting MJCF file fails to display in Genesis.

Troubleshooting Steps:

    Initially, I tried importing other models, such as 'go1', which displayed correctly.
    By comparing the XML file from the asset folder with the 'go1' file, I noticed that the URDF generated by SolidWorks, when exported to MJCF, has unique 'geom' tags representing the robot's body. In contrast, the XML files in the examples have two parts: 'visual' and 'collision'.
    Further investigation revealed that the 'contype' and 'conaffinity' attributes determine whether the model is displayed and whether collision detection is enabled.

Proposed Solution:
I copied all the 'geom' tags and set the 'contype' and 'conaffinity' attributes of the copied 'geom' to 0. This allowed the model to display correctly.
![2024-12-20 21-47-11 的屏幕截图](https://github.com/user-attachments/assets/d87e2ea2-23a5-4550-aa26-7d741bd14edd)

Is my solution appropriate? Have others encountered similar issues?

Thank you for your time and assistance.

**Comment by YunSang123** [comment]
Hi, I am a beginner in Genesis. Can you tell me how you imported your custom urdf file?

**Comment by juanjuanjuanbing** [comment]
> Hi, I am a beginner in Genesis. Can you tell me how you imported your custom urdf file?

You  can use a plug-in（https://github.com/Genesis-Embodied-AI/Genesis/issues/164#issuecomment-2571686481）on solidworks to transfer your .stl files into .urdf
Then it can be transfered into .mjcf file by MuJoCo 
`    ~/mujoco/mujoco210/bin
    ./compile /path/to/model.urdf /path/to/model.xml`

**Comment by zhouxian** [comment]
Urdf should be able to load into Genesis directly without converting to MJCF :)

**Comment by juanjuanjuanbing** [comment]
> Urdf should be able to load into Genesis directly without converting to MJCF :)

You are right, transferring to the mjcf file is the route previously trained on mujoco. Thank you for your suggestion
